docs/schema/2.0/manifest.schema.json

```diff
@@ -177,12 +164,31 @@
               "type": "string"
             }
           }
-        },
-        "publicKeys": {
-          "type": "array",
-          "items": {
-            "type": "string",
-            "format": "uri"
+        }
+      }
+    },
+    "apiS2S": {
+      "scopes": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "s2sId": {
+        "type": "string"
+      },
+      "publicKeys": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "value": {
+              "type": "string",
+              "format": "uri"
```

Comment:
uri? it will look like this
```
-----BEGIN PUBLIC KEY-----
2MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAtC+rEuKmqpdD5qvBZ2i33Hxcp7aJlV0rEiJs15n
fMUj+pPlmhPJatkQnQGTEkNDv0C9iHHyaaaNGfavgs44R+a7cAXe4qkyfVelisFAV5aOzyg1arRNkhOOjSMM
nfPmWpWAosOUUJ5wzkhCQU5V3c1AgsROTDESYCO3o992z2dHSENguJVhrs07dy5P5tqajgB09yIcqPqo0Wow
2P69ZRI6tVgvE4Iiy8d3ojOE2Ccjx0GXTbJ6C1SUbq2E0soSFY3z3TbXygJntmFerN7c2bfrCr1+nqigU2ah
QFKKnq9OJj6cZEd8HX8YUUIpBEFC7fSZkSzYzvDrSUMo98LwIDAQAB
-----END PUBLIC KEY-----
```

Reply:
My mistake. It was Friday.